.gitignore

```diff
@@ -36,3 +36,4 @@ flutter-studio/.gradle/
 flutter-idea/build/
 flutter-idea/.gradle/
 tool/plugin/.idea
+**/gradle-wrapper.jar
```

Comment:
These Gradle wrappers are not downloaded by the Flutter tool. They are used to maintain the repo.

I'm creating `.allstar/binary_artifacts.yaml` to address this issue. I'm documenting each artifact we require.